admin/app/components/solidus_admin/ui/button/component.rb

```diff
@@ -54,6 +55,8 @@ def initialize(tag: :button, text: nil, class_name: nil, size: :m, scheme: :prim
   end
 
   def call
-    content_tag(@tag, @text, class: @class_name, **@attributes)
+    content_tag(@tag, class: @class_name, **@attributes) do
+      safe_join([@icon_tag, @text].compact, " ")
+    end
```

Comment:
👀 @elia, could https://github.com/solidusio/solidus/pull/5284 possibly address this issue?